### ISSUE TITLE
Failed installation with gcc error

### ISSUE THREAD

**Issue by wisetc** [gawel/pyquery#114]
It failed for gcc error when install lxml.

> gcc: error: /usr/lib/rpm/redhat/redhat-hardened-cc1: No such file or directory
> /tmp/xmlXPathInitJJakmp.c:1:26: fatal error: libxml/xpath.h: No such file or directory

I have libxml2 installed, but it didn't help.

![picture: lxml install error](http://7xk1q0.com1.z0.glb.clouddn.com/bugfix/lxml-bug-install.png)

**Comment by jeanbaptistelab** [comment]
Do you have the libxml2-devel package installed ? The libxml2 is the runtime package, you need the "devel" version if you want to compile something that uses libxml2.

**Comment by gawel** [comment]
@jeanbaptistelab is right. you need both libxml and libxslt -devel packages.

btw pyquery is a pure python library and do not require gcc. report your problem to the appropriate place

**Comment by wisetc** [comment]
@jeanbaptistelab @gawel Thanks for your help.

I have solved the installation problem. For I'm running on fedora23, it may have yum command deprecated. So I [installed rpm-build][fedora] for it. And then I followed your advice, installed libxml2-devel and python-devel packages. Finally, I have pyquery installed.

It's not a pyquery bug indeed. Thank you.

[fedora]: https://ask.fedoraproject.org/en/question/61075/mysql_config-cflags-does-not-work-on-fedora-21/
